habitat-baselines/habitat_baselines/config/rearrange/rl_hl_srl_onav.yaml

```diff
@@ -1,12 +1,14 @@
 # @package _global_
-# Pick and place are kinematically simulated.
-# Navigation is fully simulated.
+# Pick and place are instantaneous skills, 
+# where environment is reset to final state.
```

Comment:
Reset or set? Reset usually means we restart the episode in RL.

Reply:
yep sorry, fixed this comment.